docs/client-sdk/javascript/tutorials/filter-and-label-conversations-by-id-and-metadata.md

```diff
@@ -0,0 +1,168 @@
+---
+sidebar_label: Filter and label conversations
+sidebar_position: 3
+---
+
+# Filter and label conversations using conversation IDs and metadata
+
+With XMTP, a conversation is a set of messages sent between two blockchain account addresses (addresses). Two addresses can have multiple ongoing conversations.
+
+You can assign an ID and other metadata to a conversation to use as conversation filters and labels in your app.
+
+For example, you can use conversation IDs and metadata to filter conversations by:
+
+* An app  
+Filter to display only conversations created by an app.
+
+* A message type<!--added this one because it matches the example we show in the code snippets - okay?-->  
```

Comment:
We might want to clarify the distinction between conversation IDs and Content Types. Just because you use a custom content type doesn't mean you need to use conversation IDs

Reply:
Thank you for this - adding now.